demo/src/app/components/shared/api-docs/api-docs.component.html

```diff
@@ -1,40 +1,51 @@
 <div class="api-doc-component">
-  <h2>
-    <a [routerLink]="" fragment="{{apiDocs.className}}" ngbdFragment>
+  <h2 class="title">
+    <a
+      [routerLink]=""
+      fragment="{{apiDocs.className}}" ngbdFragment
+      title="Anchor link to: {{apiDocs.className}}"
+    >
       <img src="img/link-symbol.svg" alt="Anchor link to: {{apiDocs.className}}"/>
     </a>
-    <a (click)="trackSourceClick()" href="https://github.com/ng-bootstrap/ng-bootstrap/tree/master/{{apiDocs.fileName}}" target="_blank">{{apiDocs.className}}</a>
+    <span>{{apiDocs.className}}</span>
+    <a
+      class="github-link"
+      (click)="trackSourceClick()"
+      href="https://github.com/ng-bootstrap/ng-bootstrap/tree/master/{{apiDocs.fileName}}"
+      target="_blank"
+      title="Link to Github: {{apiDocs.className}}"
+    >
+      <img src="img/github.svg" alt="Link to Github {{apiDocs.className}}"/>
+    </a>
   </h2>
-  <p>{{apiDocs.description}}</p>
-    <table class="table table-sm table-hover">
-      <tbody>
-        <tr>
-          <td class="label-cell">Selector</td>
-          <td class="content-cell"><code>{{apiDocs.selector}}</code></td>
-        </tr>
-        <tr *ngIf="apiDocs.exportAs">
-          <td class="label-cell">Exported as</td>
-          <td class="content-cell"><code>{{apiDocs.exportAs}}</code></td>
-        </tr>
-      </tbody>
-    </table>
+  <h5><small class="text-muted">Component / Directive</small></h5>
```

Comment:
Would be nice to extract the exact value from the code if later

Reply:
Yep, we can easily do this in our doc script that walks TS sources.

Reply:
We could just say `Directive` waiting for the script to be updated.